portal/src/graphql/portal/CreateProjectScreen.tsx

```diff
@@ -195,16 +174,6 @@ function CreateProjectScreenContent(props: CreateProjectScreenContentProps) {
                 "CreateProjectScreen.app-id.description"
               )}
             />
-            {isFirstProject ? (
-              <FormPhoneTextField
```

Comment:
Sorry I am not sure about this. Are we removing the field that collecting phone number of user on onboarding from portal?
@fungc-io

Reply:
Oh I guess it is because the phone number was filled in the survey already, so we will check the survey instead of the old attribute. Let me mark this resolved first.

Reply:
@tung2744 , good point, please remove it from the onboarding. I forgot about it

Reply:
@toshinari123 , the number in the onboarding can be removed